tests/unit/src/main/scala/tests/BuildServerInitializer.scala

```diff
@@ -0,0 +1,110 @@
+package tests
+
+import scala.concurrent.Future
+import scala.util.Properties
+
+import scala.meta.internal.builds.BuildTool
+import scala.meta.internal.metals.Debug
+import scala.meta.internal.metals.Messages.ImportBuild
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.internal.metals.ServerCommands
+
+import ch.epfl.scala.bsp4j.BspConnectionDetails
+import com.google.gson.Gson
+import org.eclipse.lsp4j.MessageActionItem
+
+trait BuildServerInitializer {
+  def initialize(layout: String, expectError: Boolean = false): Future[Unit]
+}
+
+trait QuickBuildInitializer extends BuildServerInitializer {
+  this: BaseLspSuite =>
+  override def initialize(
+      layout: String,
+      expectError: Boolean = false
+  ): Future[Unit] = {
+    Debug.printEnclosing()
```

Comment:
I'm not 100% sure, but this was always called before in the `initialize` Do we only want to call this here, or do we also want to for the other initializers?

Reply:
We definitely want them everywhere.